src/canvas/SelectableCanvas.ts

```diff
@@ -1423,10 +1431,10 @@ export class SelectableCanvas<
    * This is the functional part of discardActiveObject.
    * @param {Event} [e] Event (passed along when firing "object:deselected")
    * @param {Object} object the next object to set as active, reason why we are discarding this
-   * @return {Boolean} true if the selection happened
+   * @return {Boolean} true if occurred
    * @private
    */
-  _discardActiveObject(e?: TPointerEvent, object?: FabricObject) {
+  protected _discardActiveObject(e?: TPointerEvent, object?: FabricObject) {
```

Comment:
this is not protected, we should also remove the `private` in the jsdoc. There are reasons why people want to use this and i use this too.